packages/driver/src/cy/commands/waiting.ts

```diff
@@ -270,8 +281,51 @@ export default (Commands, Cypress, cy, state) => {
     })
   }
 
+  Cypress.primaryOriginCommunicator.on('wait:for:xhr', ({ args: [str, options] }, originPolicy) => {
+    options.isCrossOriginSpecBridge = true
+    waitString(null, str, options).then((responses) => {
+      // remove props that aren't serializable (these aren't used by consumers so are okay to remove)
+      const omitFn = (response) => _.omit(response, ['subscriptions', 'setLogFlag'])
```

Comment:
Shouldn't `preprocessForSerialization` omit non serializable props already?

Reply:
Yeah, it does, updated [957b80e](https://github.com/cypress-io/cypress/pull/21579/commits/957b80e549ec301eb382a3cef29a232288013c58)